### PR TITLE
Update build.sh

### PR DESCRIPTION
**Pull request by lijilong12**

### CODE REVIEW

**Comment by Memory2314** [comment]
Are you a master translator?🤣👉
I hope you can close yourself.

**Comment by ghost** [comment]
omg Pro Translator

**Comment by JohnCarl2234** [comment]
Update HAHAHA?

**Comment by ghost** [comment]
Bro think that Mandarin is useable for bash script💀💀

**Comment by hakureiyuyuko** [comment]
什么百度翻译？

**Comment by ShionKanagawa** [comment]
lmfao Pro PR

**Comment by uqsme** [comment]
出院

**Comment by JohnCarl2234** [comment]
LMAO 🤣🤣🤣 Just translating everything

**Review by uqsme — COMMENTED** [review]
神金

File: build.sh
Comment (LoveFearsHesitationMost) [inline build.sh:1]:
```bash
alias 回声="echo"
```

**Comment by ssp97** [comment]
Has your code been tested

**Comment by MukjepScarlet** [comment]
唐完了

**Pull request closed without merging**